core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -108,6 +108,7 @@ public final class SystemSessionProperties
     public static final String OPTIMIZE_DISTINCT_AGGREGATIONS = "optimize_mixed_distinct_aggregations";
```

Comment:
`This can happen if partitions by columns that would otherwise be aggregated` some words are missing